### DISCUSSION TITLE
How did you find other RSEs at your institution?

### DISCUSSION THREAD

**Discussion by jsubida** [USRSE/ag-institutional-rse-networking#2] [Q&A]
Can you provide additional context like size of your group, size of your institution? (See the [Welcome Announcement](https://github.com/USRSE/ag-institutional-rse-networking/discussions/1) for more ideas)

**Comment by l-espana** [comment]
I haven't yet 😭
I'm at a medical academic research institution of about 7,000 people. I work with/know a couple other programmers but have not yet had success getting a group together.

**Comment by jsubida** [comment]
A colleague and I are starting small and trying to find other RSEs within our (relatively large) department. Baby steps
